src/components/Card.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+class Card extends React.Component {
+
+  render() {
+    return (
+      <div className={ this.props.modifier ? 'p-card--' +  this.props.modifier : 'p-card' }>
+        <header className="p-card__header">
+          <img src={ this.props.imgSrc } alt="placeholder" />
+        </header>
+        <h3 className="p-card__title">{ this.props.title }</h3>
+        <p className="p-card__content">{ this.props.cardContent }</p>
```

Comment:
Rather than doing `cardContent` as a property you can do `{ this.props.children }` and pass the content in as child DOM:

```
<Card
  modifier="highlighted"
  title="Card title"
  imgSrc="http://placehold.it/200x32">
    Lorem ipsum dolor sit amet, consectetur adipisicing.
</Card>
```

Which also makes it easier to include markup as the child:

```
<Card
  modifier="highlighted"
  title="Card title"
  imgSrc="http://placehold.it/200x32">
    <h1>An entire site within a card</h1>
    <p>Probably an essay.</p>
    <img src="memes.gif" />
</Card>
```

Reply:
@Lukewh Thanks for all the feedback, really useful 💚